imx.mk

```diff
@@ -82,9 +82,11 @@ u-boot-defconfig: $(UBOOT_PATH)/.config
 
 .PHONY: u-boot
 u-boot: u-boot-defconfig tfa ddr-firmware
+	# Copy DDR4 firmware
 	cp $(FIRMWARE_PATH)/$(FIRMWARE_VERSION)/firmware/ddr/synopsys/lpddr4_pmu_train_*.bin \
-		$(TF_A_PATH)/build/$(TFA_PLATFORM)/release/bl31.bin \
 		$(UBOOT_PATH)
+	# Copy BL31 binary from TF-A
+	cp $(TF_A_PATH)/build/$(TFA_PLATFORM)/*/bl31.bin $(UBOOT_PATH)
```

Comment:
Please introduce `TF_A_BUILD ?= release` (or `debug`) as is done in `fvp.mk`, `hikey.mk`, `hikey960.mk` and `poplar.mk`.
TF-A is being used by a growing number of platforms so it would certainly make sense to move some macros and recipes to `common.mk`, but until that's done we should at least do things in a consistent manner.